apps/web/app/AppProviders.tsx

```diff
@@ -104,17 +105,16 @@ export default function AppProviders({ children }: AppProvidersProps) {
     }
   }, []);
 
-  const handleLogError = useCallback((err: Error) => console.error(err), []);
+  const handleLogError = useCallback((err: Error) => logger.error(err), []);
 
   useSprig(sprigEnvironmentId);
-
   return (
     <ErrorsProvider context="web">
       <CookieManagerProvider
         projectName="base_web"
         locale="en"
         region={Region.DEFAULT}
-        log={console.log}
+        log={(str, options) => logger.info(str, options)}
```

Comment:
This is the line that caused the issue last time, by wrapping it on a => function, logger will be defined correctly